omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -110,11 +114,26 @@ public String get(PageModel model,
 			Context.removeProxyPrivilege(GET_LOCATIONS);
 		}
 
+		Boolean isLocationUserPropertyAvailable = isLocationUserPropertyAvailable(administrationService);
+		Object showLocation = pageRequest.getAttribute("showSessionLocations");
+		if(showLocation != null && showLocation.toString().equals("true")) {
+			// if the request contains a attribute as showSessionLocations, then ignore isLocationUserPropertyAvailable
```

Comment:
Confising how showLocation is true and then you set is location user property available to false?

Reply:
if the URL has the parameter as "showSessionLocations" with value True, then we need to ignore the userProperty method(if exist or not). So simply setting isLocationUserPropertyAvailable=false to disable this feature to view the location selector in the login screen. (See the address bar of the browser in the attached images in the PR)

Reply:
who sets the showSessionLocations parameter to true and when?

Reply:
When a user doesn't get the location userProperty, then it will redirect to the login page with showSessionLocations=true. So I'm checking this parameter to show/hide the location selector here.  See this line - https://github.com/openmrs/openmrs-module-referenceapplication/pull/47/files#diff-bd998e9f6ec90d297ceba7fb14c2fe66R249